### DISCUSSION TITLE
Tailwind V3

### DISCUSSION THREAD

**Discussion by gaussylean** [ben-rogerson/twin.macro#585] [General] [answered]
I know it literally just came out, but does it look it would be hard to integrate v3 into the library?
https://tailwindcss.com/blog/tailwindcss-v3

**Comment by ben-rogerson** [comment, marked as answer]
There are many new features so it will take a while to bring them all into Twin.
Follow the [v3 issue](https://github.com/ben-rogerson/twin.macro/issues/589) to keep up to date with it all.
